### PR TITLE
Feature/cshrcp 323

### PR DESCRIPTION
**Pull request by frontendjim**

### CODE REVIEW

File: lib/modules/logger/index.js [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+const consoleLoggerOptions = {
+    handleExceptions: true,
+    json: false,
+    colorize: true,
+    timestamp: true,
+};
+
+const winstonLogger = new winston.Logger({
+    transports: [
+        new winston.transports.File({
```
Comment (reachsaurabh) [inline lib/modules/logger/index.js]: Doctor Winston does the job :)

Comment (frontendjim) [reply lib/modules/logger/index.js]: I like the fact there's an option to format errors in JSON... makes it easy to read if it's run through a parser. :)

File: app.js [hunk 2]
```diff
@@ -9,51 +9,27 @@ const sassMiddleware = require('node-sass-middleware');
...
-            appender: 'issues',
-            level: 'error',
-        },
-    },
-    categories: {
-        default: { appenders: ['just-errors', 'everything'], level: 'debug' },
-    },
-});
-const logger = log4js.getLogger();
+if (!fs.existsSync('./logs')) {
```
Comment (reachsaurabh) [inline app.js]: log folder should be externalized , is it within project structure , looks like ./

Comment (frontendjim) [reply app.js]: Yep currently within the project... I'm assuming we're using that cloud based wizardry soon aren't we?

**Pull request merged**